gphoto2/Makefile.am

```diff
@@ -103,35 +106,22 @@ gphoto2_webapi_SOURCES      += range.c range.h
 gphoto2_webapi_SOURCES      += mongoose/mongoose.c mongoose/mongoose.h
 gphoto2_webapi_SOURCES      += server.c server.h
 
+gphoto2_webapi_CPPFLAGS     += -DPROGNAME=\"gphoto2-webapi\"
+gphoto2_webapi_CPPFLAGS     += -DGPHOTO2_WEBAPI
+
+# FIXME: Does interactive config make sense for gphoto2-webapi?
 if HAVE_CDK
 gphoto2_webapi_SOURCES      += gphoto2-cmd-config.c gphoto2-cmd-config.h
+gphoto2_webapi_CFLAGS       += $(CDK_CFLAGS)
+gphoto2_webapi_LDADD        += $(CDK_LIBS)
 endif
 
+# FIXME: Does ASCII art preview make sense for gphoto2-webapi?
```

Comment:
@thorsten-l Does ASCII art preview make sense for gphoto2-webapi?

Reply:
not really ... but streaming liveview same as capture-movie perhaps

Reply:
Like that server you telnet to and it shows some star wars intro as ASCII...

Reply:
i also think this does'nt make sense

Reply:
Ah. Your commit 6128ee97d4383c59093e0939aa87af0b9d272390 has already removed the code from `gphoto2-webapi` which needs aalib. So the `if HAVE_AA ... endif` part of `Makefile.am` can now be removed (for gphoto2_webapi_* only of course).